services/datcore-adapter/src/simcore_service_datcore_adapter/utils/client_base.py

```diff
@@ -56,7 +56,7 @@ def _create_instance() -> None:
         api_cls.create_once(
             app,
             client=httpx.AsyncClient(
-                http2=True, base_url=api_baseurl, timeout=api_general_timeout
+                http2=False, base_url=api_baseurl, timeout=api_general_timeout
```

Comment:
add the note from the description in the code. That will explain why we explicitly disable it. 
http2=False is the default, right?